.eslintrc.js

```diff
@@ -49,6 +50,8 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 'off'
```

Comment:
I'm not sure we should disable this. Did you try https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems ?

Reply:
@akmadian

Reply:
Well, disabling it allows devs using windows OS to contribute directly on the project without having to run anything in the command line, or adding a folder. It'll be particularly useful to newbies who want to contribute without having to be confused just after running `npm run serve`, Instead we'll all be able to focus on the actual code we should be contributing. Or I could just open a new issue to adjust readme of this repo directing windows users on what to actually do after cloning the repo.

Reply:
Yep, I'm on Windows and I have to add the rule to disable this every time I'm on a new branch, or wait for all the errors to get printed before the page actually gets served up.

Seems that there's no cross platform linebreak style in eslint, at least not that I could find. I don't think it's a good idea to mix styles, but if we choose one or the other, devs on either unix or windows systems will have to adapt. I'm not really sure what we should do about this. Thoughts @brenoferreira?

Reply:
I think it's better to disable linting with the `webpack-dev-server` and have that integrated only on text editors. That way you won't have issues on the terminal when you run `npm run dev`.

We run that during CI build, so it shouldn't be a problem. Plus people should run it before pushing code to Github.

Reply:
We should also set the appropriate settings on .gitattributes on the root dir
https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems